cmd/reports.go

```diff
@@ -37,7 +37,39 @@ var reportsListCmd = &cobra.Command{
 	},
 }
 
+var reportsDownloadCmd = &cobra.Command{
+	Use:   "download",
+	Short: "Download Reports",
+	Long:  `Download reports in different formats such as CSV, excel or PDF`,
+}
+
+var reportsDownloadCsvCmd = &cobra.Command{
+	Use:   "summary-csv",
+	Short: "List Vulnerability Analysis Reports",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("file id is required")
```

Comment:
The error message should be `Report ID is required`

Reply:
Updated

Reply:
You list reports using file id not the report id

Reply:
But this is not list command. It is `download summary-csv` command 😛

Reply:
My bad. This is corrected